components/query_filter/utils_unittest.cc

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/components/query_filter/utils.h"
+
+#include "testing/gtest/include/gtest/gtest.h"
+#include "third_party/abseil-cpp/absl/types/optional.h"
+#include "url/gurl.h"
+
+TEST(BraveQueryFilter, FilterQueryTrackers) {
```

Comment:
Might be good to add a test using POST like `EXPECT_FALSE(https://brave.com/?glicd=123, ..., POST, false)`

Reply:
Sure like I said I should add some more tests. But just need to learn a little bit about c++ testing on brave-core.

Reply:
Added these tests